ext/symengine/symengine.c

```diff
@@ -57,6 +57,9 @@ void Init_symengine()
     // Sympify as a Module Level Function
     rb_define_module_function(m_symengine, "convert", cutils_sympify, 1);
     rb_define_global_function("SymEngine", cutils_sympify, 1);
+    
+    // evalf as a Module Level Function
+    rb_define_module_function(m_symengine, "evalf", cutils_evalf, 3);
```

Comment:
Maybe it's easier if you did it in pure ruby and you rename this method to something like `_evalf` which can be called from `evalf` pure ruby method.